local.tf

```diff
@@ -66,7 +66,7 @@ locals {
     launch_template_version           = "$Latest"                                # The lastest version of the launch template to use in the autoscaling group
     launch_template_placement_tenancy = "default"                                # The placement tenancy for instances
     launch_template_placement_group   = ""                                       # The name of the placement group into which to launch the instances, if any.
-    root_encrypted                    = ""                                       # Whether the volume should be encrypted or not
+    root_encrypted                    = false                                    # Whether the volume should be encrypted or not
```

Comment:
I think setting this to `null` is safer?

Reply:
Reverted

Reply:
It should be a boolean value I guess. `encrypted` in a bool variable.

failure logs:

```
Error: Incorrect attribute value type

  on .terraform/modules/tidb-operator.eks/workers.tf line 188, in resource "aws_launch_configuration" "workers":
 188:     encrypted = lookup(
 189:       var.worker_groups[count.index],
 190:       "root_encrypted",
 191:       local.workers_group_defaults["root_encrypted"],
 192:     )
    |----------------
    | count.index is 0
    | local.workers_group_defaults["root_encrypted"] is ""
    | var.worker_groups is tuple with 1 element

Inappropriate value for attribute "encrypted": a bool is required.
```

terraform version: 

```
$ terraform version
Terraform v0.12.23
```

Reply:
Ok, let's go with bool. Thoughts @max-rocket-internet ??

Reply:
Updated PR: #766